ext/standard/tests/file/lstat_stat_variation15.phpt

```diff
@@ -33,11 +33,11 @@ symlink($filename, $linkname);
 // checking lstat() and stat() on links
 echo "*** Testing lstat() on a link after changing its access permission ***\n";
 clearstatcache();
+sleep(1);
```

Comment:
The sleep should be one line further down: After the first lstat, but before the chmod.